app/auth/modules/User.js

```diff
@@ -0,0 +1,26 @@
+class User {
+  static create(id, name = '', isAdmin = false) {
+    localStorage.setItem('id', JSON.stringify(id));
+    localStorage.setItem('name', JSON.stringify(name));
+    localStorage.setItem('isAdmin', JSON.stringify(isAdmin));
+  }
```

Comment:
Why do you need JSON.stringify on strings id , name ??

Reply:
`localStorage` stores only strings, so it is needed to convert an object into a string before storing it in `localStorage`.

Reply:
That u understood, bt isnt id and name already a string

Reply:
Yeah, but it is better to use for all otherwise we will have to keep track of which is a string and which is not. Here, we can use a uniform rule of converting every object to `JSON` before storing it in local storage.